src/memray/commands/run.py

```diff
@@ -215,6 +219,12 @@ def prepare_parser(self, parser: argparse.ArgumentParser) -> None:
             help="Record allocations in child processes forked from the tracked script",
             default=False,
         )
+        parser.add_argument(
+            "--trace-python-allocators",
```

Comment:
I wonder if we should shorten this to `--trace-pymalloc` or even just `--pymalloc`? `--trace-python-allocators` is a lot to remember/type.

I almost wonder if we should just call it `--leaks`, since the time when it's most useful is in conjunction with `memray flamegraph --leaks`, but that's probably too reductive...

I think `--trace-pymalloc` might be the best possible name, on balance.

Reply:
Technically `pymalloc` != Python allocators. You could track the python allocators where they in turn do not call pymaalloc (they could call something else like malloc).

Reply:
I see your point, but worry that no one will use the option if we call it `--trace-python-allocators` 😄